README.md

```diff
@@ -2,7 +2,14 @@
 
 This repo contains a large file uploader app and a theme for Liferay's Zendesk Help Center.
 
+## Large File Uploader App
+
+A simple app that shows up in the agent view that displays a link to an external large file uploader page that allows the user to upload a file greater than the Zendesk attachment file size limit.
+
 ## Theme
+
+The Guide theme for Liferay's Help center.
```

Comment:
Capitalize `Center` to keep it consistent with other references to `Liferay's Help Center`.

Reply:
Should this also say `Liferay's Zendesk Help Center` to be consistent with everything?

Reply:
:ok_hand: